### ISSUE TITLE
A problem of the concatenate function in utlis.py

### ISSUE THREAD

**Issue by an17-21** [EricGJB/DL-based-signal-detection#2]
```
def DetectNet(lr,input_shape,filter_num,lstm_units,kernel_size,drop_ratio,lstm_drop_ratio,dense_units):
    # model = utils.DetectNet(0.0003,(2,128),60,128,10,0.2,0.2,120)
    '''
        Network architecture of DetectNet
    '''
    ConvInput = Input(input_shape)
    x1 = Convolution1D(filter_num,kernel_size,padding='same',data_format='channels_first',activation='relu')(ConvInput)
    x1 = Dropout(rate=drop_ratio)(x1)
    x2 = Convolution1D(filter_num,kernel_size,padding='same',data_format='channels_first',activation='relu')(x1)
    x2 = Dropout(rate=drop_ratio)(x2)
    x3 = Flatten()(x2)
    # dimension reduction
    x4 = Dense(input_shape[-1],activation='linear')(x3)
    x4 = Reshape(target_shape=(1,input_shape[-1]))(x4)
    
    LSTMInput = concatenate([x4,ConvInput],axis=1)
    
    y1 = LSTM(units=lstm_units,return_sequences=True,recurrent_dropout=lstm_drop_ratio,input_shape=(input_shape[-1],3))(LSTMInput)
    y2 = LSTM(units=lstm_units,dropout=lstm_drop_ratio)(y1) 
    y2 = Flatten()(y2)
    y3 = Dense(dense_units,activation='relu')(y2)
    
    predictions = Dense(2, activation='softmax')(y3)
    model = Model(inputs=ConvInput,outputs=predictions)
    model.compile(loss='categorical_crossentropy',optimizer=Adam(lr=lr), metrics=['accuracy'])
    model.summary()
    return model
```
When the code `LSTMInput = concatenate([x4,ConvInput],axis=1)` is executed, the shape of ConvInput is (2,128) and the shape of x4 is (1,128). How can they be concatenated in axis=1?
